drivers/overlay/ostweaks_linux.go

```diff
@@ -21,10 +22,43 @@ func writeSystemProperty(key, value string) error {
 	return ioutil.WriteFile(path.Join("/proc/sys", keyPath), []byte(value), 0644)
```

Comment:
Personally I would have make the following change:
1) create a satisfy condition function type.  
2) enhance the sysctlConf map with string and struct
3) cleanup the logic

Example for the whole logic: https://play.golang.com/p/vgsQ455Lvdf

Reply:
Think I've got it sorted now. PTAL